1-general-examples/call-automation-api-from-mainframe/README.md

```diff
@@ -0,0 +1,377 @@
+<h1>Call Automation API from Mainframe </h1>
+
+## Description
+
+This example shows how to be able to connect to a Control-M (or Helix Control-M) environment from Mainframe (z/OS) and run Automation API services from JCL.
```

Comment:
The following example demonstrates how to connect to Control-M or Helix Control-M and run Automation API using JCL from Mainframe (z/OS).